<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeedleCalibrationWidget</class>
 <widget class="QWidget" name="NeedleCalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeedleCalibrationWidget</string>
  </property>
  <widget class="QSlider" name="framesSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Frames</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frameDisplay">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>300</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>1800</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay</string>
   </property>
  </widget>
  <widget class="QSlider" name="delaySlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="delayDisplay">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calibrateBt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>framesSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NeedleCalibrationWidget</receiver>
   <slot>numberOfFrames(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delaySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NeedleCalibrationWidget</receiver>
   <slot>delay(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrateBt</sender>
   <signal>clicked()</signal>
   <receiver>NeedleCalibrationWidget</receiver>
   <slot>calibrate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBt</sender>
   <signal>clicked()</signal>
   <receiver>NeedleCalibrationWidget</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>numberOfFrames(int)</slot>
  <slot>delay(int)</slot>
  <slot>calibrate()</slot>
  <slot>save()</slot>
 </slots>
</ui>
